src/perception/tag_detector/tag_detector.hpp

```diff
@@ -64,6 +81,8 @@ namespace mrover {
         TimeProfiler mProfiler;
 
         void onInit() override;
+                
+        void publish_thresh(cv::Mat const& mImg);
```

Comment:
Parameter should be lowercase. The `m<whatever>`  syntax is used for [m]ember variables of a class.

Reply:
I'm gonna delete this param cuz mImg is a class member variable updated by the pcCallback